docs/pages/foundations/overview.js

```diff
@@ -0,0 +1,38 @@
+// @flow strict
+import { type Node } from 'react';
+import { Flex } from 'gestalt';
+import COMPONENT_DATA from '../../docs-components/COMPONENT_DATA.js';
+import Page from '../../docs-components/Page.js';
+import PageHeader from '../../docs-components/PageHeader.js';
+import IllustrationCard from '../../docs-components/IllustrationCard.js';
+import IllustrationSection from '../../docs-components/IllustrationSection.js';
+import IllustrationContainer from '../../docs-components/IllustrationContainer.js';
+
+export default function FoundationsOverview(): Node {
+  return (
+    <Page title="Foundations overview" hideSideNav hideEditLink>
+      <Flex direction="column" width="100%">
+        <IllustrationContainer justifyContent="start">
+          <PageHeader
+            name="Foundations overview"
+            description="Here you will find the foundational elements for designing UI components, surfaces and patterns. These include graphic elements like iconography, color and typography to structural. They also cover elevation, layout and accessibility."
+            type="guidelines"
+          />
+        </IllustrationContainer>
+        <IllustrationSection title="" grid="auto-fill" min={312}>
+          {COMPONENT_DATA.foundations.map((element, idx) => (
+            <IllustrationCard
+              headingLevel={2}
+              key={idx}
+              href={element?.path ?? `/web/${element.name.replace(/\s/g, '_').toLowerCase()}`}
```

Comment:
I feel like this that `convertToUrl` function we keep using everywhere? 

Should we redefine it here again, so it's all aligned, and then in a future update, we can refactor the code and store that function in a util file somewhere.

Reply:
Yeah, we should cleanup some of this if we reuse them.We can do  a cleanup follow up PR